website/blog/2023-08-01-announcing-materialized-views.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Introducing Materialized Views and how to use them"
```

Comment:
In general we try and avoid framing dev blog posts as _announcements_. Can we frame this around why people would find these useful?

Reply:
How about- 

`We have Materialized Views: What do we do now?` 

or

`How to not fuck up Materialized Views` --> maybe too on the nose?

Reply:
What about - `A new tool to optimize your dbt workflow: Materialized Views?`

or `Optimizing Materialized Views for the dbt workflow`?

Speed and performance don't feel like the right call here but making MVs is the right call. Like I view dbt as the guardrails you need for MVs